.github/workflows/test.yml

```diff
@@ -24,13 +24,5 @@ jobs:
         run: sudo apt-get update && sudo apt-get install mesa-opencl-icd ocl-icd-opencl-dev
       - name: Check out code
         uses: actions/checkout@v1
-      - name: Cache dependencies
```

Comment:
No need for this, since the self-hosted runner already has all dependencies cached.